scripts/module/money_drop.nut

```diff
@@ -0,0 +1,30 @@
+alive <- array(100, null);
+MoneyPickups <- {};
+
+function onPlayerSpawn( player ) {
+  alive[player.ID] = true;
+  player.Cash = 500;
+}
+
+function onPlayerPart( player, reason ) {
+  alive[player.ID] = null;
+}
+
+function onPlayerHealthChange( player, lastHP, newHP ) {
+  if ( newHP < 0.1 && alive[player.ID]) {
```

Comment:
Use an epsilon, which is smaller `1e-6` or something. Maybe squirrel has an epsilon defined as part of the language?